init-and-run-wiki

```diff
@@ -12,7 +12,26 @@ fi
 
 if [ ! -d /var/lib/tiddlywiki/mywiki ]; then
   /usr/bin/env node $NODEJS_V8_ARGS $tiddlywiki_script mywiki --init server
+
+  mkdir /var/lib/tiddlywiki/mywiki/tiddlers
 fi
 
-exec /usr/bin/env node $NODEJS_V8_ARGS $tiddlywiki_script mywiki --server 8080 $:/core/save/all text/plain text/html ${USERNAME:-user} ${PASSWORD:-'wiki'} 0.0.0.0
+
+# Configure the tiddlywiki to be served on a path if it is given
+WEBHOST_TID=/var/lib/tiddlywiki/mywiki/tiddlers/\$__config_tiddlyweb_host.tid
+if [ -n "${SERVE_URI}" ]
```

Comment:
Would you mind adopting the `if .. then` style used in the rest of the script? That would be more consistent. 

```suggestion
if [ -n "${SERVE_URI}" ]; then
```